content/en/install.md

```diff
@@ -3,101 +3,206 @@ title: Installation
-- [conda](#pip-install)
-- [Package Manager](#package_manager)
-- [Source](#source)
+- [Project-based (e.g. `uv`, `pixi`)](#project-based) (recommended for new users)
+- [Environment-based (e.g. `pip`, `conda`)](#environment-based) (the traditional workflow)
+- [System package managers](#system-package-managers) (not recommended)
+- [Building from source](#building-from-source) (for debugging and development)
 
-Methods differ in ease of use, coverage, maintenance of old versions,
-system-wide versus local environment use, and control. With pip or
-Anaconda\'s conda, you can control the package versions for a specific
-project to prevent conflicts. Conda also controls non-Python packages,
-like MKL or HDF5. System package managers, like `apt-get`, install
-across the entire computer, often have older versions, and don\'t have
-as many available versions. Source compilation is much more difficult
-but is necessary for debugging and development. If you don\'t know which
-installation method you need or prefer, we recommend the Scientific
-Python Distribution [Anaconda](https://www.anaconda.com/download/).
+To install SciPy with [static type stubs],
+see [Installing with type stubs](#type-stubs).
 
-<a name="distributions"></a>
+[static type stubs]: https://typing.readthedocs.io/en/latest/guides/libraries.html
 
-# Scientific Python Distributions (recommended)
+<a name="project-based"></a>
 
-Python distributions provide the language itself, along with the most
-commonly used packages and tools. These downloadable files require
-little configuration, work on almost all setups, and provide all the
-commonly used scientific Python tools.
+## Project-based workflows
 
-[Anaconda](https://www.anaconda.com/download/) works on Windows, Mac,
-and Linux, provides over 1,500 Python packages, and is used by over 15
-million people. Anaconda is best suited to beginning users; it provides
-a large collection of libraries all in one.
+### Installing with `uv`
 
-For more advanced users who will need to install or upgrade regularly,
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge) is a more
-suitable way to install the `conda` (and `mamba`, a faster `conda` alternative)
-package manager.
+{{< admonition tip >}}
+If it is your first time installing a Python library, start here!
+{{< /admonition >}}
 
-Other options include:
+Here is a step-by-step guide to setting up a project to try SciPy, with [`uv`],
+a Python package manager.
 
-- [WinPython](https://winpython.github.io): Another free distribution
-  including scientific packages and the Spyder IDE; Windows only, but
-  more actively maintained and supports the latest Python 3 versions.
-- [Pyzo](https://pyzo.org): A free distribution based on Anaconda
-  and the IEP interactive development environment; Supports Linux,
-  Windows, and Mac.
+[`uv`]: https://docs.astral.sh/uv/
 
-<a name="pip-install"></a>
+1. Install `uv`, following [the instructions in the `uv` documentation][install-uv].
```

Comment:
@zanieb I wonder whether the instructions in the `uv` docs could be made even more beginner-friendly? Perhaps for people who aren't familiar with using a terminal, where you say "use `curl`", you could instead say something like "use `curl` in a terminal prompt", and maybe link out to an explanation of how to get a terminal/shell on different OSs?